engine/src/snapshot/logic.rs

```diff
@@ -87,7 +87,7 @@ pub enum WriteError {
 pub fn encrypt_content<O: Write>(plain: &[u8], output: &mut O, key: &Key) -> Result<(), WriteError> {
     let work_factor = age::RECOMMENDED_MINIMUM_ENCRYPT_WORK_FACTOR;
     // `work_factor = 1` is intentionally just for development.
-    // let work_factor = 1;
+    let work_factor = 1;
```

Comment:
Undo

Reply:
Yeah, always forget about it. Undid commit.